ui/components/app/snaps/snap-ui-markdown/snap-ui-markdown.js

```diff
@@ -23,7 +23,8 @@ const Paragraph = (props) => (
     variant={TextVariant.bodyMd}
     className="snap-ui-markdown__text"
     overflowWrap={OverflowWrap.Anywhere}
-    color={TextColor.inherit}
+    color={TextColor.textDefault}
```

Comment:
Why can't we inherit anymore ?

Reply:
We need to inherit to support certain cases where the color is overridden by the parent

Reply:
Reverted!